srcpkgs/jmol/template

```diff
@@ -0,0 +1,28 @@
+# Template for 'jmol'
+pkgname=jmol
+version=14.29.17
+revision=1
+noarch=yes
+maintainer="Brenton Horne <brentonhorne77@gmail.com>"
+homepage="http://jmol.sourceforge.net/"
+depends="virtual?java-environment"
+license="LGPL-3"
+hostmakedepends="unzip"
+short_desc="An open-source Java/JavaScript-based molecule viewer"
+distfiles="${SOURCEFORGE_SITE}/projects/jmol/files/Jmol/Version%20${version%.*}/Jmol%20${version}/Jmol-${version}-binary.tar.gz http://jmol.sourceforge.net/images/Jmol_icon_128.png"
```

Comment:
one link per line, see checksum=, also distfiles= should be directly above checksum=

Reply:
Is it OK the way it is now, with only skip_extraction between distfiles and checksum?